packages/boot/test/bootstrapTests/vat-orchestration.test.ts

```diff
@@ -95,7 +96,7 @@ test('makeAccount returns an ICA connection', async t => {
   t.regex(remoteAddress, /icahost/);
   t.regex(localAddress, /icacontroller/);
   t.regex(chainAddress.address, /cosmos1/);
-  t.regex(chainAddress.chainId, /FIXME/); // TODO, use a real chainId #9063
+  t.is(chainAddress.chainId, 'somechain-1');
```

Comment:
`somechain-1` isn't a real chainId in the sense of the fetched chain registry data. I guess that's OK?
This test clearly doesn't actually require a real chainId.